src/Type/WPEnumType.php

```diff
@@ -13,12 +13,14 @@
 class WPEnumType extends EnumType {
 
 	/**
-	 * WPInputObjectType constructor.
+	 * WPEnumType constructor.
 	 *
 	 * @param array $config
 	 */
 	public function __construct( $config ) {
-		$config['name']   = ucfirst( $config['name'] );
+		$name           = ucfirst( $config['name'] );
+		$config['name'] = apply_filters( "graphql_typename_$name", $name, $config, $this );
```

Comment:
I've been trying to limit the number of dynamic variable names. It's more difficult to document and harder to debug. 

I'd prefer a static filter name such as `graphql_type_name` and then use the arguments to determine whether the callback should be applied to the filter.